Defining_Policies_and_Profiles/master.adoc

```diff
@@ -0,0 +1,16 @@
+= Defining Policies and Profiles
+CloudForms Documentation Team <{product-title}-docs@redhat.com>
```

Comment:
this doesn't seem right as it's an email address.  Also, I don't think the master.adoc files are supposed to come over like this anyway. @elladeon ?